src/lib/addons/slack-app.ts

```diff
@@ -52,17 +53,20 @@ export default class SlackAppAddon extends Addon {
         parameters: ISlackAppAddonParameters,
     ): Promise<void> {
         try {
-            const { accessToken } = parameters;
+            const { accessToken, defaultChannels } = parameters;
```

Comment:
I'm wondering if we should opt-in on the events we send to the default channels, I don't think all events are relevant for Slack and it can potentially affect our rate limiting. I know default channels are optional, but I'm just wondering if it's a good thing to do.

Reply:
Ok, the events received by handlEvent are already filtered here: https://github.com/Unleash/unleash/blob/34c4dd573ac5081419609128ac64e6d6d8830f24/src/lib/services/addon-service.ts#L114